example/App.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { UniversalPortal } from '../';
+
+export default () => (
+  <React.Fragment>
```

Comment:
There's a shorter syntax available for fragments: https://reactjs.org/docs/fragments.html#short-syntax